### PR TITLE
Flip forward camera. Close #21

### LINKED ISSUE #21: Front facing camera invert
The front facing camera should have its image inverted across the y axis.

### PR DESCRIPTION
**Pull request by ianstenbit**

### CODE REVIEW

**Comment by ianstenbit** [comment]
Please wait until after PR #24 is merged to merge this.

**Review by jakecarlson1 — APPROVED** [review]
Much more natural. Our users will like this.

**Pull request merged**
